### PR TITLE
Adds Beta Mode Banner

### PR DESCRIPTION
**Pull request by qwwqwwq**

### CODE REVIEW

File: winterfell/webapp/src/templates/root.pug [hunk 1]
```diff
@@ -1,3 +1,6 @@
+if showBanner
+    div#banner
+        h3 #{bannerMessage}
```
Comment (ahsu1230) [inline winterfell/webapp/src/templates/root.pug]: h3 might be too big, we can stick to h5

File: winterfell/config/bannerConfig.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+/**
+ * Dynamically configure the root page banner
+ */
+module.exports = {
+  showBanner: true,
+  bannerMessage: "Hello! Vetafi.org is in beta, and not yet ready for public use. To learn more email us at info@vetafi.com"
```
Comment (ahsu1230) [inline winterfell/config/bannerConfig.js]: We can remove the "Hello!" part

**Comment by qwwqwwq** [comment]
You're right, h5 is perfect

You really have a great eye for design :)

**Comment by ahsu1230** [comment]
lol thanks lucky guess

**Pull request merged**